src/authentication/logging/LogConfiguration.js

```diff
@@ -10,6 +12,8 @@ class LogConfiguration {
     loggingLevel;
     maxLogFiles;
     enableMasking;
+    externalLogger;
```

Comment:
Change `externalLogger` to `hasExternalLogger` and change `externalLoggerObj` to `externalLogger`